client/signup/jetpack-connect/install-step.jsx

```diff
@@ -7,20 +7,42 @@ import React from 'react';
  * Internal dependencies
  */
 import Card from 'components/card';
+import { preventWidows } from 'lib/formatting';
 
 export default React.createClass( {
 	displayName: 'JetpackInstallStep',
 
+	renderText() {
+		const { action } = this.props;
+		let { text } = this.props;
+
+		if ( ! action ) {
+			return (
+				<span>
+					{ preventWidows( text ) }
+				</span>
+			);
+		}
+
+		return (
+			<span>
+				{ text }
+				<span> </span>
```

Comment:
I guess this is for forcing a whitespace between text and action, isn't it? I think this would be better done by css instead of adding a new node to the DOM... suggestion, @rickybanister  ?